packages/react-dom/src/__tests__/ReactDOMHooks-test.js

```diff
@@ -102,9 +102,11 @@ describe('ReactDOMHooks', () => {
     );
 
     inputRef.current.value = 'abc';
-    inputRef.current.dispatchEvent(
-      new Event('input', {bubbles: true, cancelable: true}),
-    );
+    await act(() => {
```

Comment:
`act` is needed because this is the experimental channel, which no longer flushes updates in the legacy root after this change unless the flag is on, which seems fine since we're following up to remove the APIs unless the flag is on anyway?